universal-application-tool-0.0.1/app/controllers/FileController.java

```diff
@@ -46,7 +48,9 @@ public FileController(
               if (!fileKey.contains(String.format("applicant-%d", applicantId))) {
                 return notFound();
               }
-              return redirect(storageClient.getPresignedUrlString(fileKey));
+              return redirect(
+                  storageClient.getPresignedUrlString(
+                      URLDecoder.decode(fileKey, StandardCharsets.UTF_8)));
```

Comment:
The encoding/decoding is only changing "/" -> "2f%" from what I can tell. We had to add this because the URL was wasn't being recognized because everything that was supposed to be "/" was instead "2f%"

Reply:
Even though it works both ways consider decoding it before doing anything else so that all current and future logic here is dealing with the decoded version

Reply:
Done.